dts/bindings/spi/st,stm32-spi.yaml

```diff
@@ -0,0 +1,42 @@
+---
+title: STM32 SPI
+id: st,stm32-spi
+version: 0.1
+
+description: >
+    This binding gives a base representation of the STM32 SPI controller
+
+inherits:
+    !include spi.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "st,stm32-spi"
+
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+    interrupts:
+      type: array
+      category: required
+      description: required interrupts
+      generation: define
+
+    interrupt-names:
+      type: stringlist
+      category: optional
+      description: readable string describing the interrupts
+      generation: define
+
+    clocks:
```

Comment:
Isn't this redundant since this inherits spi.yaml?

Reply:
Removed.